Makefile

```diff
@@ -23,10 +23,11 @@ checks:
 	@sleep 3
 	@echo "$(OK_COLOR)==> Running checks tests against container deps $(NO_COLOR)" && \
 		HEALTH_GO_PG_DSN="postgres://test:test@`docker-compose port postgres 5432`/test?sslmode=disable" \
-		HEALTH_GO_MQ_DSN="amqp://guest:guest@`docker-compose port rabbit 5672`/" \
+		HEALTH_GO_MQ_DSN="http://guest:guest@`docker-compose port rabbit 15672`/" \
```

Comment:
why `http`? it is amqp, not the http

Reply:
it uses Rabbit HTTP API to do healthchecks, probably I need to rename because it is not real DSN anymore